FirebaseABTesting/Sources/ABTConstants.h

```diff
@@ -33,3 +33,11 @@ static NSString *const kABTEventDictionaryNameKey = @"name";
 static NSString *const kABTEventDictionaryOriginKey = @"origin";
 static NSString *const kABTEventDictionaryParametersKey = @"parameters";
 static NSString *const kABTEventDictionaryTimestampKey = @"timestamp";
+
+#pragma mark - Errors
+
+static NSString *const kABTErrorDomain = @"com.google.abt";
+
```

Comment:
nit: If this is a public header, you should add a doc comment to both the declaration and the value. If not ignore this comment.

Reply:
This isn't a public header.